website/docs/cdktf/cli-reference/commands.mdx

```diff
@@ -353,7 +353,7 @@ Options:
       --local                     Use local state storage for generated Terraform.                                                                    [boolean] [default: false]
       --cdktf-version             The cdktf version to use while creating a new project.                                                             [string] [default: "0.0.0"]
       --from-terraform-project    Use a terraform project as the basis, CDK constructs will be generated based on the .tf files in the path                             [string]
-      --enable-crash-reporting    Enable crash reporting for the CLI, see https://www.terraform.io/cdktf/telemetry#crash-reporting for more details                    [boolean]
+      --enable-crash-reporting    Enable crash reporting for the CLI, refer to https://www.developer.hashicorp.com/cdktf/telemetry#crash-reporting for more details                    [boolean]
```

Comment:
Yeah, I can add something. We're free to choose.
There is already [`/telemetry`](https://github.com/hashicorp/terraform-cdk/blob/0e1a06803f681dcb6a453b8f0f0e2bc84b60a805/cdk.tf/vercel.json#L155-L156) (which seems like it would need to be updated..), so maybe let's use `/crash-reporting`?

Reply:
Also: https://developer.hashicorp.com/cdktf/telemetry#crash-reporting does not seem to exist yet..

Reply:
pushed a commit adding `/crash-reporting`